### ISSUE TITLE
Add BrowserStack Certificates

### ISSUE THREAD

**Issue by waleedsadek-panx** [PanXProject/awesome-certificates#24]
Fork this repo, add free certificates from this link: https://test-university.browserstack.com/catalog/learning-paths to this repo and submit a pull request.

**Comment by Kundai10** [comment]
@waleedsadek-panx Is this issue available? I can do it. Also, is this a hacktoberfest issue?

**Comment by waleedsadek-panx** [comment]
Yes all the open issues are available and this one particular one is part of hacktober

**Comment by Kundai10** [comment]
@waleedsadek-panx May I please have more context on this issue?
